.github/workflows/validate-packages.yaml

```diff
@@ -0,0 +1,19 @@
+name: Validate package manifests
+
+on:
+  pull_request:
+
+jobs:
+  validate-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+      - name: Install json validate
+        run: go install github.com/santhosh-tekuri/jsonschema/cmd/jv@latest
```

Comment:
This command will download and compile the jv package for every run. Maybe we can cache it? Also, I would suggest using a stable version instead of latest.

Reply:
migrated to [cardinalby/schema-validator-action](https://github.com/cardinalby/schema-validator-action) and tested the validation here: https://github.com/glasskube/packages/actions/runs/7613451277/job/20733367403